dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-email/src/main/java/org/apache/dolphinscheduler/plugin/alert/email/MailSender.java

```diff
@@ -405,7 +405,7 @@ public void deleteFile(File file) {
      */
     private void handleException(AlertResult alertResult, Exception e) {
         log.error("Send email to {} failed", receivers, e);
-        alertResult.setMessage("Send email to {" + String.join(",", receivers) + "} failed，" + e.toString());
+        alertResult.setMessage("Send email to {" + String.join(",", receivers) + "} failed，" + e.getMessage());
```

Comment:
```suggestion
        alertResult.setMessage("Send email to {" + String.join(",", receivers) + "} failed:" + e.getMessage());
```

Reply:
You can keep consistency and use `String.format`.

Reply:
Done, Please check again